### PR TITLE
feat(kit): support `--tui-accordion-item-gap` for `accordion`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?

01-accordion-gap snap

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by splincode** [comment]
@waterplea @nsbarsukov @vladimirpotekhin hello, please review me

File: projects/demo/src/modules/components/accordion/accordion.template.html [hunk 1]
```diff
@@ -80,6 +81,19 @@
                         </div>
                     </ng-template>
                 </tui-accordion-item>
+                <tui-accordion-item
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/accordion/accordion.template.html:84]: Sorry, don't understand the necessity of this new css-variable.

User has direct access to `tui-accordion-item`. Could it set all required margins by yourself ?

Comment (splincode) [reply projects/demo/src/modules/components/accordion/accordion.template.html:84]: difficult 

https://stackblitz.com/edit/angular-a1bcwx?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.less

Comment (waterplea) [reply projects/demo/src/modules/components/accordion/accordion.template.html:84]: Maybe we can make it not difficult, instead of expanding the API?

Comment (splincode) [reply projects/demo/src/modules/components/accordion/accordion.template.html:84]: sure

**Pull request closed without merging**